.devcontainer/postCreate.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -x
+
+conda init bash
```

Comment:
looks like this script is executed after the container initializes. wouldn't it be better to move these commands to the Dockerfile?

I think that's why I encountered this weird behavior:

- once VSCode opened in my browser, I ran `conda env list` and I couldn't see the jupysql env, but then after 1-2 minutes I ran it again and it showed up

Reply:
So the repo is mounted as volume in the container at run-time.

This means you can't run the `pkgmt setup` command below in the Dockerfile as the repo specific config won't be available at container build time. This is exactly what the `postCreateCommand` is meant to alleviate.

I think it's possible to add the `conda init` and `conda create` commands in the Dockerfile and this might speed up the container startup time, but we'd still need this script to run `pkgmt setup`. IMO there's something to be said to keep all setup instructions together in a single script, this is how I often see it done. I haven't timed it, but my guess is also that these conda commands only take seconds - negligible compared to `pkgmt setup`.  

> once VSCode opened in my browser, I ran conda env list and I couldn't see the jupysql env, but then after 1-2 minutes I ran it again and it showed up

Normally VSCode shows an indicator in the terminal window that it's still running the postCreate command. I was  referring to this when I wrote in the docs "After the codespace has finished setting up", but l understand this might not be clear. I'll try to capture a screenshot for this and add it in the docs.